Packages/com.unity.inputsystem/InputSystem/Plugins/UI/InputSystemUIInputModule.cs

```diff
@@ -2221,6 +2234,8 @@ private void FilterPointerStatesByType()
                     // We have touch or tracked input. Kill mouse/pen pointer, if we have it.
                     for (var i = 0; i < m_PointerStates.length; ++i)
                     {
+                        ref var state = ref GetPointerStateForIndex(i);
```

Comment:
Is this line superfluous?

Reply:
This is indeed superfluous, it's a remnant from a previous change that I deemed unnecessary in the end.